models/party.js

```diff
@@ -21,8 +21,15 @@ class Party {
 
     const newParty = new Parse.Object("Party");
     newParty.set("name", body.name)
-    const query = new Parse.Query("User");
-    const dm = await query.get(body.dm.objectId);
+    var dm
+    try {
+      const query = new Parse.Query("User");
+      dm = await query.get(body.dm.objectId);
+    }
+    catch {
+      throw new NotFoundError("Invalid userId")
```

Comment:
Is it possible to add the objectID in the error?  It helps for debugging in the future when you log the error to database.

For example

throw new NotFoundError(`Invalid userId: ${body.dm.objectId}`)